src/platform/site-wide/cta-widget/CallToActionAlert.jsx

```diff
@@ -7,6 +7,8 @@ export default function CallToActionAlert({
   alertText,
   buttonText,
   buttonHandler,
+  secondaryLinkText,
```

Comment:
I think we should rename the first `buttonHandler` and `buttonText` to `primaryButton*`.

Reply:
The link is actually a button because it performs some action when clicked, so that should probably be renamed to reflect that.